devel/ci/bodhi-ci

```diff
@@ -861,8 +894,16 @@ def _build_jobs_list(
     else:
         buffer_output = concurrency != 1 and len(releases) != 1
     jobs = []
+    if command in ('all', 'integration') and 3 in pyvers:
+        integration_deps_build_jobs = [
+            IntegrationBuildJob(app_name="resultsdb", release="f29", buffer_output=buffer_output),
+            IntegrationBuildJob(app_name="waiverdb", release="f29", buffer_output=buffer_output),
+            IntegrationBuildJob(app_name="greenwave", release="f29", buffer_output=buffer_output),
```

Comment:
Is the release meaningful on these dependencies? If not, should we make it an optional parameter?

Reply:
It's not meaningful at the moment, but those deps are the only place where the release argument isn't meaningful. And there are quite a few places in bodhi-ci that expect `self.release` to be set.
Not sure it's worth making it optional.